addons/point_of_sale/static/src/js/models.js

```diff
@@ -660,33 +660,31 @@ exports.PosModel = Backbone.Model.extend({
         this.set('cashier', user);
         this.db.set_cashier(this.get('cashier'));
     },
-    //creates a new empty order and sets it as the current order
+    // creates a new empty order and sets it as the current order
     add_new_order: function(){
         var order = new exports.Order({},{pos:this});
         this.get('orders').add(order);
         this.set('selectedOrder', order);
         return order;
     },
-    // load the locally saved unpaid orders for this session.
+    // load the locally saved unpaid orders for this PoS Config.
     load_orders: function(){
         var jsons = this.db.get_unpaid_orders();
         var orders = [];
         var not_loaded_count = 0;
 
         for (var i = 0; i < jsons.length; i++) {
             var json = jsons[i];
-            if (json.pos_session_id === this.pos_session.id) {
+            if (json.lines.length > 0) {
                 orders.push(new exports.Order({},{
                     pos:  this,
                     json: json,
                 }));
-            } else {
-                not_loaded_count += 1;
             }
         }
 
         if (not_loaded_count) {
-            console.info('There are '+not_loaded_count+' locally saved unpaid orders belonging to another session');
+            console.info('There are '+not_loaded_count+' locally saved unpaid orders belonging to another config');
```

Comment:
`not_loaded_count` will always be 0 here (you removed the increment); either get rid of this warning (pbly the simplest thing to do as you fixed its reason for being there) or make it work (more complex since you'd need to check localstorage for other config)

Reply:
Indeed, this is never shown now. To check it we would have to open other local storages, like you pointed out yourself before if I'm not mistaking. This would be to much work and not really useful.